scripts/setRequests-v3validator.ts

```diff
@@ -0,0 +1,314 @@
+    },
+    // IN corner
+
+    {
+      requestId: 450 * requestIdModifier,
+      schema: schema,
+      claimPathKey: schemaClaimPathKey,
+      operator: 4,
+      value: [...new Array(64).fill(0)],
+      slotIndex,
+      queryHash,
+      circuitIds,
+      allowedIssuers,
+      skipClaimRevocationCheck,
+      claimPathNotExists: claimPathDoesntExist
+    },
+    // NIN corner
+    {
+      requestId: 550 * requestIdModifier,
+      schema: schema,
+      claimPathKey: schemaClaimPathKey,
+      operator: 5,
+      value: [...new Array(63).fill(0), 19960424],
+      slotIndex,
+      queryHash,
+      circuitIds,
+      allowedIssuers,
+      skipClaimRevocationCheck,
+      claimPathNotExists: claimPathDoesntExist
+    },
+    // NE corner
+    {
+      requestId: 650 * requestIdModifier,
+      schema: schema,
+      claimPathKey: schemaClaimPathKey,
+      operator: 6,
+      value: [...new Array(64).fill(0)],
+      slotIndex,
+      queryHash,
+      circuitIds,
+      allowedIssuers,
+      skipClaimRevocationCheck,
+      claimPathNotExists: claimPathDoesntExist
+    }
+  ];
+
+  try {
+    for (let i = 0; i < ageQueries.length; i++) {
+      const query = ageQueries[i];
+      console.log(query.requestId);
+
+      const operatorKey =
+        Object.keys(QueryOperators)[Object.values(QueryOperators).indexOf(query.operator)];
+
+      query.queryHash = calculateQueryHash(
+        query.value,
+        query.schema,
+        query.slotIndex,
+        query.operator,
+        query.claimPathKey,
+        claimPathDoesntExist
+      ).toString();
+
+      const invokeRequestMetadata = {
+        id: '7f38a193-0918-4a48-9fac-36adfdb8b542',
+        typ: 'application/iden3comm-plain-json',
+        type: 'https://iden3-communication.io/proofs/1.0/contract-invoke-request',
+        thid: '7f38a193-0918-4a48-9fac-36adfdb8b542',
+        body: {
+          reason: 'for testing',
+          transaction_data: {
+            contract_address: erc20verifierAddress,
+            method_id: 'b68967e2',
+            chain_id: 80001,
+            network: 'polygon-mumbai'
+          },
+          scope: [
+            {
+              id: query.requestId,
+              circuitId: circuitIdV3,
+              query: {
+                allowedIssuers: ['*'],
+                context: schemaUrl,
+                credentialSubject: {
+                  birthday: {
+                    [operatorKey]:
+                      query.operator === Operators.IN || query.operator === Operators.NIN
+                        ? query.value
+                        : query.value[0]
+                  }
+                },
+                type: type
+              }
+            }
+          ]
+        }
+      };
+
+      const tx = await erc20Verifier.setZKPRequest(query.requestId, {
+        metadata: JSON.stringify(invokeRequestMetadata),
+        validator: validatorAddressV3,
+        data: packValidatorParams(query)
```

Comment:
packV3ValidatorParams

Reply:
I suggest to rename packValidatorParams to packV2ValidatorParams so it would be explicit which version you should use.